Description: Disable termion backend for packaged version is too low
 Required 4.x, packaged 1.x
Author: Blair Noctis <ncts@debian.org>
Forwarded: not-needed
Last-Update: 2025-03-04
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -100,3 +100,3 @@
 
-[dependencies.termion]
+[disabled.dependencies.termion]
 version = "4"
@@ -138,3 +138,3 @@
 #]
-termion-backend = ["dep:termion"]
+#termion-backend = ["dep:termion"]
 toml = ["cursive_core/toml"]
